beetmoverscript/src/beetmoverscript/utils.py

```diff
@@ -123,9 +121,6 @@ def generate_beetmover_template_args(context):
     task = context.task
     release_props = context.release_props
 
-    if is_maven_action(context.action):
```

Comment:
Nit: not sure we can remove this function as it's used here too - https://github.com/mozilla-releng/scriptworker-scripts/blob/master/beetmoverscript/src/beetmoverscript/task.py#L25 in order to check the schema validation.

Reply:
Come to think about it, I'm surprised tests passed with this function being removed. I would've assumed https://github.com/mozilla-releng/scriptworker-scripts/blob/master/beetmoverscript/tests/test_task.py#L33 to fail.

Reply:
Indeed https://firefox-ci-tc.services.mozilla.com/tasks/U8VagEHyQ1CG2pi88P5_eg/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FU8VagEHyQ1CG2pi88P5_eg%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L142 seems to have passed; that's odd 🤔

Reply:
But is it? The function is still there (L87), it's just this one call that I removed, which would always return false in that context.

Reply:
Don't mind me, I misread the code initially and changed it locally without double-checking again in the PR. This looks good!